evalml/data_checks/invalid_target_data_check.py

```diff
@@ -115,12 +116,12 @@ def validate(self, X, y):
             ...                             'num_null_rows': 2,
             ...                             'pct_null_rows': 50.0},
             ...                 'code': 'TARGET_HAS_NULL'}],
-            ...     'actions': [{'code': 'IMPUTE_COL',
+            ...     'actions': {"action_list": [{'code': 'IMPUTE_COL',
             ...                  'data_check_name': 'InvalidTargetDataCheck',
             ...                  'metadata': {'columns': None,
             ...                               'rows': None,
             ...                               'is_target': True,
-            ...                               'impute_strategy': 'mean'}}]}
+            ...                               'impute_strategy': 'mean'}}], "default_action": None}}
```

Comment:
Another mega nit: can the "default_action" go on a new line for readability? (Same applies in a couple other places, but this is the most confusing one I think)

Reply:
Done!